mvpa2/support/afni/lib_prep_afni_surf.py

```diff
@@ -166,14 +183,13 @@ def augmentconfig(c):
     if c['sid'] is None:
         print"Warning: no subject id specified"
 
-
     c['prefix_sv2anat'] = 'SurfVol2anat'
 
     # update steps
     if c.get('steps', 'all') == 'all':
-        c['steps'] = 'toafni+mapico+moresurfs+skullstrip+align+makespec+makespecboth+makesurfmasks'
-
-
+        c[
+            'steps'] = ('toafni+mapico+moresurfs+skullstrip+'
```

Comment:
yikes

BTW -- by PEP8ing I have meant only the code which was touched by the fix... but ok since work is done already ;)